docs/migration_guide.rst

```diff
@@ -1,6 +1,38 @@
 Migration guide
 ===============
 
+******************************
+0.24.0 Migration guide
+******************************
+
+Version 0.24.0 of **starknet.py** comes with support for signing with Ledger wallet! It also introduces devnet client to interact with additional features of the Starknet Devnet.
+
+0.24.0 Targeted versions
+------------------------
+
+- Starknet - `0.13.1.1 <https://docs.starknet.io/documentation/starknet_versions/version_notes/#version0.13.1.1>`_
+- RPC - `0.7.1 <https://github.com/starkware-libs/starknet-specs/releases/tag/v0.7.1>`_
+
+0.24.0 Breaking changes
+-----------------------
+
+.. currentmodule:: starknet_py.net.client_models
+
+1. :class:`CompiledContract` and :class:`ContractClass` have been renamed to :class:`DeprecatedCompiledContract` and :class:`DeprecatedContractClass`.
+2. :class:`ContractClassSchema` have been renamed to :class:`DeprecatedContractClassSchema`
+
+0.24.0 Minor changes
+-----------------------
+
+.. currentmodule:: starknet_py.devnet_utils.devnet_client
+
+1. :class:`DevnetClient` has been implemented to interact with additional features of the Starknet Devnet
+
+.. currentmodule:: starknet_py.net.signer.ledger_signer
+
+2. :class:`LedgerSigner` has been implemented to sign with Ledger hardware wallet
```

Comment:
It is a bit ambiguous what `minor changes` mean, but I would define it as changes in the existing code that don't break anything. Therefore I would remove `DevnetClient` and `LedgerSigner` from this section. It seems sufficient to mention it above and in the release notes.